tests/test_replica.py

```diff
@@ -288,6 +304,31 @@ def test_delete_replicas_from_datasets_old(self, rse_factory, mock_scope, root_a
 
         assert [f for f in list_files(scope=mock_scope, name=tmp_dsn2)] == []
 
+    def test_refresh_replicas(self, rse_factory, mock_scope, root_account):
+        """REPLICA (CORE): Refresh replicas"""
+        _, rse_id = rse_factory.make_mock_rse()
+        nbfiles = 5
+
+        # add 5 replicas
+        files = [{'scope': mock_scope, 'name': did_name_generator('file'), 'bytes': 1, 'adler32': '0cc737eb', 'meta': {'events': 10}} for _ in range(nbfiles)]
+        add_replicas(rse_id=rse_id, files=files, account=root_account, ignore_availability=True)
+        # adding a repilca will set updated_at time to unix epoch, so we expect is set
+        r = {'scope': files[0]['scope'], 'name': files[0]['name'], 'rse_id': rse_id}
+        original = get_replica_updated_at(r)
+        assert original is not None
+
+        # we update the state to BEING_DELETED
+        update_replica_state(rse_id, r['scope'], r['name'], ReplicaState.BEING_DELETED)
+        original = get_replica_updated_at(r)
+
+        time.sleep(2)
+        ok = refresh_replicas(rse_id, [r])
+        assert ok is True
+
+        got = get_replica_updated_at(r)
+        assert got > original
```

Comment:
nit: I'd rename some of these variables to have clearer names:
`r` -> `replica_dict`
`ok` -> maybe just `assert refresh_replicas(rse_id, [r])`
`original` -> `original_timestamp`
`got` -> `updated_timestamp`